Dockerfile.flodgatt

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:18.04
+
+# Create the mastodon user
+ARG UID=991
+ARG GID=991
+RUN apt update && \
+	echo "Etc/UTC" > /etc/localtime && \
+	apt install -y whois wget && \
+	addgroup --gid $GID mastodon && \
+	useradd -m -u $UID -g $GID -d /opt/mastodon mastodon && \
+	echo "mastodon:`head /dev/urandom | tr -dc A-Za-z0-9 | head -c 24 | mkpasswd -s -m sha-256`" | chpasswd && \
```

Comment:
Wouldn't it be easier to just lock the account?

Reply:
Feel free to test and PR after this is merged... I'm leaving this for now